src/main/java/org/jboss/xavier/analytics/pojo/input/VMWorkloadInventoryModel.java

```diff
@@ -0,0 +1,179 @@
+package org.jboss.xavier.analytics.pojo.input;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+
+
+public class VMWorkloadInventoryModel
+{
+    //common/name
+    private String provider;
+    //common/ems_clusters/v_parent_datacenter
+    private String datacenter;
+    //common/ems_clusters/name
+    private String cluster;
+    //vms/name
+    private String vmName;
+    //sum of vms/hardware/disks/size_on_disk
+    private BigDecimal diskSpace;
+    //vms/ram_size_in_bytes
+    private Integer memory;
+    //vms/num_cpu
+    private Integer cpuCores;
+    //vms/operating_system/product_name
+    private String osProductName;
+    //hardware/guest_os_full_name
+    private String guestOSFullName;
+    //vms/has_rdm_disk
+    private boolean hasRdmDisk;
+    //count of nics object within the vms/hardware
+    private Integer nicsCount;
+    //hardware/disks/filename
+    private Collection<String> vmDiskFilenames;
+    private Collection<Collection<Map<String, String>>> systemServices;
```

Comment:
This is the structure : 
],
            "system_services": [
              {
                "id": 9669,
                "name": "rhnsd",
                "typename": "linux_initprocess"
              },
So unless it's dynamic I would suggest having a Class representing the fields

Reply:
I'll change this to be 'systemServicesNames',  a collection of Strings containing all the different 'name' values, which is the only field actually required by the rules.